packs/book_of_earth_pack/xml/boe_katas.xml

```diff
@@ -15,7 +15,7 @@
       <Requirements field="crab_hida_bushi_school" type="tag">Hida Bushi School</Requirements>
       <Requirements field="crab_hiruma_scout_school" type="tag">Hiruma Scout School</Requirements>
       <Requirements field="phoenix_shiba_bushi_school" type="tag">Matsu Berserker School</Requirements>
-      <Requirements field="crane_daidoji_iron_warrior" type="tag">Daidoji Iron Warrior</Requirements>
+      <Requirements field="crane_daidoji_iron_warrior_school" type="tag">Daidoji Iron Warrior</Requirements>
```

Comment:
This is not right, the Daidoji Iron Warrior's school id is actually 'crane_daidoji_iron_warriors' (check the core pack). It doesn't follow the "_school" convention probably by mistake, but we cannot change it (or we break all Daidoji Iron Warrior character saves)

Reply:
Reverted